README.md

```diff
@@ -76,12 +88,43 @@ pip install -r requirements.txt
 Export the DB Configuration and turn on Debugging if necessary
 
 ```shell script
+# MySQL
+export THR_DB_NAME=thr_users  # The DB should already be created
 export DEBUG=1
 export THR_DB_NAME=thr_db  # The DB should already be created
 export THR_DB_USER=user
 export THR_DB_PASSWORD=password
 export THR_HOST=localhost
 export THR_PORT=3306
+# Elasticsearch
+export ES_HOST=localhost:9200
+```
+
+Download and run Elasticsearch (follow the installation steps on [Elasticsearch website](https://www.elastic.co/downloads/elasticsearch))
+
+You can (optional but recommended) download and run Kibana too ([installation steps](https://www.elastic.co/downloads/kibana)).
+
+Migrate and Rebuild ES index
+
+```shell script
+python manage.py migrate
+python manage.py search_index --rebuild -f
+```
+
+The last command will create an index called `trackhubs` (the index schema is still WIP), we can get the list of indices using the command
+
+```shell script
+curl -XGET "http://localhost:9200/_cat/indices"
+```
+
+To enrich the trackhubs indexed in ES and already existed in MySQL DB we can run the command
```

Comment:
You might need to be a bit more specific on what "enrich" means.

Reply:
Done, I hope it's clearer now.